source/windows/troubleshooting.markdown

```diff
@@ -122,6 +122,8 @@ Instead, wrap the builtin in `cmd.exe`:
       path => 'c:\windows\system32;c:\windows'
     }
 
+Keep in mind that in 32-bit versions of Puppet, you may be subject to file system redirection (where system32 is switched to sysWOW64 automatically by Windows). We have ways of dealing with the [redirection](/reference/lang_windows_file_paths.html#file-system-redirection-when-running-32-bit-puppet-on-64-bit-windows).
+
```

Comment:
It's not quite right. You'll want `/puppet/latest/reference/lang_windows_file_paths.html#file-system-redirection-when-running-32-bit-puppet-on-64-bit-windows`